pypit/scripts/run_pypit.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+
+
+"""
+This script pushes a FITS file to ginga
+"""
+from __future__ import (print_function, absolute_import, division,
+                        unicode_literals)
+
+try:
+    from xastropy.xutils import xdebug as debugger
+except:
+    import pdb as debugger
+
+def parser(options=None):
```

Comment:
Actually, that is built-in with argparse (try it).
I wish it would show that when you gave a bad input.
Actually, now that I think of it, I know how if you really want it.